Mage.Sets/src/mage/cards/m/MyojinOfCrypticDreams.java

```diff
@@ -0,0 +1,78 @@
+package mage.cards.m;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.EntersBattlefieldAbility;
+import mage.abilities.common.SimpleActivatedAbility;
+import mage.abilities.condition.common.CastFromHandSourcePermanentCondition;
+import mage.abilities.costs.common.RemoveCountersSourceCost;
+import mage.abilities.decorator.ConditionalOneShotEffect;
+import mage.abilities.effects.common.CopyTargetSpellEffect;
+import mage.abilities.effects.common.counter.AddCountersSourceEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.SubType;
+import mage.constants.SuperType;
+import mage.constants.TargetController;
+import mage.counters.CounterType;
+import mage.filter.FilterSpell;
+import mage.filter.predicate.Predicate;
+import mage.game.Game;
+import mage.game.stack.StackObject;
+import mage.target.TargetSpell;
+
+import java.util.UUID;
+
+/**
+ * @author Alex-Vasile
+ */
+public class MyojinOfCrypticDreams extends CardImpl {
+
+    private static final FilterSpell permanentSpellFilter = new FilterSpell("permanent spell you control");
+    static {
+        permanentSpellFilter.add(TargetController.YOU.getControllerPredicate());
+        permanentSpellFilter.add(MyojinOfCrypticDreamsPredicate.instance);
+    }
+
+    public MyojinOfCrypticDreams(UUID ownderId, CardSetInfo setInfo) {
+        super(ownderId, setInfo, new CardType[]{CardType.CREATURE}, "{5}{U}{U}{U}");
+
+        this.addSuperType(SuperType.LEGENDARY);
+        this.subtype.add(SubType.SPIRIT);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(3);
+
+        // Myojin of Cryptic Dreams enters the battlefield with an indestructible counter on it if you cast it from your hand.
+        this.addAbility(new EntersBattlefieldAbility(new ConditionalOneShotEffect(
+                new AddCountersSourceEffect(CounterType.INDESTRUCTIBLE.createInstance()),
+                CastFromHandSourcePermanentCondition.instance, ""
+        ), "{this} enters the battlefield with an indestructible counter on it if you cast it from your hand"));
+
+        // Remove an indestructible counter from Myojin of Cryptic Dreams:
+        // Copy target permanent spell you control three times. (The copies become tokens.)
+        Ability ability = new SimpleActivatedAbility(
```

Comment:
I can't seem to activate this.

Reply:
Are you holding priority after casting a permanent spell?

Reply:
Yes. I cast Cryptic Dreams from my hand, then I cast another permanent (in this case Myojin of Grim Betrayal). After paying Grim Betrayal's cost I have priority and can play instants or activate abilities. If I click on Myojin of Cryptic Dreams nothing happens. It also isn't highlighted.

Reply:
I’ll take a look in the morning, my brain is pretty fried right now

Reply:
See for example Myojin of Blooming Dawn, same issue. I have cast it, but I'm unable to activate it's ability.

<img width="581" alt="Screen Shot 2022-02-09 at 11 13 14 PM" src="https://user-images.githubusercontent.com/48962821/153335892-26e6675b-2091-464d-96ac-e7f24557ddfb.png">

Reply:
My bad, it turns out I messed up the etb ability. I fixed it in 8010ce50e4